package.json

```diff
@@ -28,54 +28,54 @@
     "@ledgerhq/hw-transport-node-hid-noevents": "6.27.1"
   },
   "dependencies": {
-    "@babel/runtime": "^7.14.8",
-    "@ledgerhq/hw-transport": "6.27.1",
+    "@babel/runtime": "^7.24.7",
+    "@ledgerhq/hw-transport": "6.30.6",
     "@ledgerhq/hw-transport-u2f": "^5.36.0-deprecated",
-    "@vue/cli-plugin-typescript": "^5.0.8",
-    "@vue/cli-service": "^5.0.8",
-    "bech32": "^1.1.4",
+    "@zondax/ledger-js": "^0.10.0",
+    "bech32": "^2.0.0",
     "buffer": "^6.0.3",
     "crypto": "^1.0.1",
     "crypto-browserify": "^3.12.0",
     "ripemd160": "^2.0.2",
-    "stream-browserify": "^3.0.0",
-    "vue": "^2.7.7",
-    "vue-template-compiler": "^2.7.10"
+    "stream-browserify": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.18.10",
-    "@babel/core": "^7.18.10",
-    "@babel/node": "^7.18.10",
-    "@babel/plugin-transform-runtime": "^7.18.9",
-    "@babel/preset-env": "^7.18.9",
-    "@ledgerhq/hw-transport-node-hid": "6.27.1",
-    "@ledgerhq/hw-transport-webusb": "6.27.1",
-    "@swc/core": "^1.2.237",
-    "@types/jest": "^26.0.24",
-    "@types/ledgerhq__hw-transport": "4.21.4",
-    "@types/node": "^18.11.9",
-    "@typescript-eslint/eslint-plugin": "^5.33.1",
-    "@typescript-eslint/parser": "^5.33.1",
+    "@babel/cli": "^7.24.7",
```

Comment:
I think it is time we stop using babel and webpack.
Once we use ts, we can remove those..